README.md

```diff
@@ -892,6 +893,19 @@ The majority (54%) are using public transit. This includes buses, trams and metr
 cadaver, cauliflower, cabbage (vegetable), catalpa (tree) and Cailleach.
 >
 ```
+#### Numa
```

Comment:
These commands need more explanation than just "Numa". Otherwise, users will not know why they should run them or what to do afterwards.

cc @bmtwl